packages/frontend/src/pages/Templates/index.tsx

```diff
@@ -1,47 +1,92 @@
-import { Center, Flex, Hide, Image, Text } from '@chakra-ui/react'
-import { Badge, Link } from '@opengovsg/design-system-react'
+import { useQuery } from '@apollo/client'
+import { AbsoluteCenter, Box, Center, Flex, Grid, Text } from '@chakra-ui/react'
+import { Link, Tile } from '@opengovsg/design-system-react'
 
-import templatesPreviewImg from '@/assets/templates-preview.svg'
 import Container from '@/components/Container'
-import NavigationDrawer from '@/components/Layout/NavigationDrawer'
+import PageTitle from '@/components/PageTitle'
+import PrimarySpinner from '@/components/PrimarySpinner'
 import * as URLS from '@/config/urls'
+import { Template } from '@/graphql/__generated__/graphql'
+import { GET_TEMPLATES } from '@/graphql/queries/get-templates'
+import { TemplateIcon } from '@/helpers/flow-templates'
+
+const TEMPLATES_TITLE = 'Templates'
 
 export default function Templates(): JSX.Element {
+  const { data, loading } = useQuery(GET_TEMPLATES)
+  const templates: Template[] = data?.getTemplates
+
+  // Sanity check if our templates file is missing
+  if (!templates || templates.length === 0) {
+    return (
+      <Box position="relative" h="100%">
+        <AbsoluteCenter>
+          <Text>
+            There are no templates now, please contact support@plumber.gov.sg
+          </Text>
+        </AbsoluteCenter>
+      </Box>
+    )
+  }
+
   return (
-    <Container w={{ base: '90%', md: '60%', lg: '50%' }}>
-      <Hide above="sm">
-        <NavigationDrawer />
-      </Hide>
-      <Center mt={{ base: '1rem', sm: '3rem' }}>
-        <Image src={templatesPreviewImg} alt="template-preview" />
-      </Center>
+    <Container
+      py={9}
+      pl={{ base: '2rem', xl: '3.5rem' }}
+      pr={{ base: '2rem', xl: '8.5rem' }}
```

Comment:
can standardize across pages

Reply:
ok as discussed, i have logged it on linear and will do it after this gets released to avoid rebase hell